### ISSUE TITLE
Balmoral Gift, Report from Khagan's Palace

### ISSUE THREAD

**Issue by geoffbon** [lensvol/fl-time-keeper#4]
Would it be possible to have the current alert for this removed if it has already been picked up in another browser?  I use a mobile browser some of the time, and if I pick up gift from Balmoral, for example, it still shows the alert to pick it up.

Your other "Small Mercies" extension seems able to read the character properties to get favour values, could this extension do something similar to read the "A Gift from Balmoral" and "A Report from the Khagan's Palace" qualities?

**Comment by lensvol** [comment]
Probably! It is an exciting idea, thank you. The only caveat I can see is that the timer itself must be hidden until you pick it up in _this_ browser. This is due to the fact that the FL server does _not_ store the exact moment you choose the "pickup" branch.

If another browser is using the same extension and you are logged into some kind of account that shares your browser preferences between instances on different devices, then it should be able to see it.

**Comment by geoffbon** [comment]
Yes, I figured the timer would be impossible, but at least it would be possible to change the message to show it had been picked up, and possibly just remove the timer portion?  

The other browser is a mobile browser, which in this case doesn't support extensions, unfortunately..

**Comment by lensvol** [comment]
That seems like a good idea, I will look into it!
